log-collector-script/linux/eks-log-collector.sh

```diff
@@ -316,6 +300,18 @@ get_common_logs() {
           tail -c 10M /var/log/messages > "${COLLECT_DIR}"/var_log/messages
           continue
         fi
+        if [[ "${entry}" == "containers" ]]; then
+          cp --force --recursive /var/log/containers/aws-node* "${COLLECT_DIR}"/var_log/
+          cp --force --recursive /var/log/containers/coredns-* "${COLLECT_DIR}"/var_log/
+          cp --force --recursive /var/log/containers/kube-proxy* "${COLLECT_DIR}"/var_log/
+          continue
+        fi
+        if [[ "${entry}" == "pods" ]]; then
+          cp --force --recursive /var/log/pods/kube-system_aws-node* "${COLLECT_DIR}"/var_log/
+          cp --force --recursive /var/log/pods//kube-system_coredns* "${COLLECT_DIR}"/var_log/
+          cp --force --recursive /var/log/pods/kube-system_kube-proxy* "${COLLECT_DIR}"/var_log/
+          continue
+        fi
```

Comment:
Thanks, this makes sense. We get the names of all pods running in `get_docker_info`, but we only need the full logs for these ones.